sandbox-open-imis/charts/db/templates/db.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBInstance
+metadata:
+  name: open-imis-db
```

Comment:
Why not use variables from values file here as well like from deployment?

Reply:
Applied suggestion.